tests/keras/backend/backend_test.py

```diff
@@ -1323,6 +1323,23 @@ def test_resize_images(self):
             K.resize_images(K.variable(xval), 2, 2,
                             data_format='channels_middle')
 
+    @staticmethod
+    def _helper_bilinear(height_factor, width_factor):
+        x_shape = (2, 3, 4, 5)
+        for data_format in ['channels_first', 'channels_last']:
```

Comment:
Suggest to make `data_format` as argument of `_helper_bilinear`. Then parameterize `test_resize_images_bilinear` with `data_format`. Otherwise, you may not be able to test both `data_format` with `pytest.raises`.